plugins/modules/azure_rm_webapp.py

```diff
@@ -856,6 +930,9 @@ def __init__(self):
         self.supported_linux_frameworks = ['ruby', 'php', 'python', 'dotnetcore', 'node', 'java']
         self.supported_windows_frameworks = ['net_framework', 'php', 'python', 'node', 'java', 'dotnetcore']
 
+        self.managed_identity = {"identity": ManagedServiceIdentity,
```

Comment:
This definition cannot be used here, because it will cause the following error, which can be detected by using 'ansible-test sanity plugin/modules/azure_rm_webapp.py". Thank you!
```e.g.
import-error: Exception attempting to import module for argument_spec introspection, 'name 'ManagedServiceIdentity' is not defined
```
```suggestion
        self.managed_identity = {"identity": ManagedServiceIdentity,
```

Reply:
Thanks.  I didn't understand the lint error.  I changed it to use a @poperty